src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const ROBOT_VALUES = Object.values(robot);
+
+  function findDuplicates(arry) {
```

Comment:
The 'findDuplicates' function should be defined outside of the 'inverseRobot' function to avoid redefining it every time the 'inverseRobot' function is called, also change arry to array